docs/architecture.md

```diff
@@ -33,34 +39,44 @@ Then, you must also rewrite all interaction between your application and its wor
 Next, you must build infrastructure to operate and scale the worker servers.
 Finally, you must operate and scale the orchestration server and its underlying data store (for example, Cassandra for Temporal), which are both single points of failure for your application.
 
+DBOS is simpler because it **only takes responsibility for checkpointing and recovering your workflows**.
```

Comment:
```suggestion
DBOS is simpler because it runs entirely **in-process**, taking responsibility only for checkpointing and recovering your workflows.
```